File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03738/p03738_num7_parsed.py
import sys

def compare_magnitudes(A, B):
    if A > B:
        return "GREATER"
    elif A < B:
        return "LESS"
    else:
        return "EQUAL"

A = int(sys.stdin.readline())
B = int(sys.stdin.readline())

result = compare_magnitudes(A, B)
print(result)

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p03738/p03738_num7_parsed.cpp
#include <iostream>
#include <string>
#include <algorithm>

int main() {
    std::string A, B;
    std::cin >> A >> B;

    if (A.length() > B.length()) {
        std::cout << "GREATER";
    } else if (A.length() < B.length()) {
        std::cout << "LESS";
    } else {
        if (A > B) {
            std::cout << "GREATER";
        } else if (A < B) {
            std::cout << "LESS";
        } else {
            std::cout << "EQUAL";
        }
    }

    return 0;
}